Paper Input Box Fix

Deantwo opened the pull request:
- Created a copy of stripped_input called stripped_multiline_input that
  does the same, expect it is a multiline input.
- Makes the writing input box multi-line again.
- Renaming paper now uses the stripped_input.
- Fixes https://github.com/tgstation/-tg-station/issues/7132

Deantwo committed:
Paper Input Box Fix

- Created a copy of stripped_input called stripped_multiline_input that
does the same, expect it is a multiline input.
- Makes the writing input box multi-line again.
- Renaming paper now uses the stripped_input.
- Fixes https://github.com/tgstation/-tg-station/issues/7132

Deantwo commented:
May have a mistake in my change to paper's rename function.
Closing this PR until I have done some more testing tomorrow.

Anyone know where "sanitize" is located by the way?

tkdrg commented:
https://github.com/tgstation/-tg-station/blob/master/code/__HELPERS/text.dm#L48
Git grep is your friend (and so is #coderbus)

Deantwo committed:
Paper Name Limit

- Added arguments to the stripped_input call.
-- usr, no idea why it worked without this.
-- MAX_NAME_LEN, so you can't make paper with super long names.

Deantwo commented:
That should fix the little bug I had made by not having added the correct maximum length.

Only issue that rename is that; if you name a paper something, then open the rename input box again and click *cancel*, the name is removed.
But as far as I know that was true before too. Seems to be a problem with the [input proc itself](http://www.byond.com/docs/ref/info.html#/proc/input). It returns null when cancelled and when the input box is left empty.

Miauw62 commented:
then add a check for the case when the input is null or empty (```if(!input || input == "")```)

Deantwo commented:
> then add a check for the case when the input is null or empty (if(!input || input == ""))

But you need to be able to remove the label again if you want.
And I can't think of a way to do that without adding a new context-menu option.

Miauw62 commented:
oh the name being removed is an actual feature, i thought it just displayed "paper -" as the name then.
this is good :V
